src/hide_if_sponsored.js

```diff
@@ -60,14 +58,14 @@ function getVisibleText(e) {
   }
   const children = e.querySelectorAll(":scope > *");
   if (children.length !== 0) {
-    if (e.style.display === "flex") {
```

Comment:
let's create a `const style = window.getComputedStyle(e);` and use it for these

Reply:
done

Reply:
Oh my bad. I thought these two are the same variable, but the second one is for the children's style.